src/IssueViz/OpenInIde/OpenInIdeHandlerImplementation.cs

```diff
@@ -108,17 +109,40 @@ internal class OpenInIdeHandlerImplementation : IOpenInIdeHandlerImplementation
             visualization = visualizationProcessor.HandleConvertedIssue(visualization) ;
         }
         issueSelectionService.SelectedIssue = visualization;
-        
+
+        toolWindowService.Show(toolWindowId);
+
         var navigationResult = navigator.TryNavigatePartial(visualization);
-        HandleNavigationResult(navigationResult, toolWindowId, visualization);
+        if (navigationResult != NavigationResult.OpenedLocation)
```

Comment:
Positive conditions should be used to make it fluent to read => swap if&else

Reply:
Done